### PR TITLE
🔥 🔥 🔥 🔥 🔥 Everything is on 🔥 🔥 🔥 🔥 🔥 

### PR DESCRIPTION
**Pull request by thebritican**
@zendesk/volunteer 

So. We did everything. The whole app is great.

But actually:
- [x] firebase hookup for locations page
- [x] firebase hookup for location page
- [x] fixed some routing
- [x] refactored services -> locations
- [x] utilize new nested firebase schema for locations
- [x] tiny css fix w/ toggles (a _bad_ solution, z-index is 💩)

great app. much fun. 🔥 

plz +1

selfie-0
[_GitHub Selfies_](https://github.com/thieman/github-selfies/)

### CODE REVIEW

**Comment by cpovey** [comment]
Ha, accidentally clicked "Close PR" button thinking it was "Cancel" for the comment I was writing. I need to go home.

**Comment by craig-day** [comment]
@cpovey how dare you!

**Comment by craig-day** [comment]
:+1:

File: components/Layout/Navigation.css [hunk 1]
```diff
@@ -18,4 +18,5 @@
   right: 0;
   padding: 8px 0 4px;
   border-bottom: 1px solid #DDD;
+  z-index: 999;
```
Comment (craig-day) [inline components/Layout/Navigation.css]: 😢

Comment (thebritican) [inline components/Layout/Navigation.css]: it's a hack for now haha we'll fix it promise

Comment (roseleaf) [inline components/Layout/Navigation.css]: promise? make followup issue?

Comment (thebritican) [inline components/Layout/Navigation.css]: we made it... on a whiteboard 😆

Comment (roseleaf) [inline components/Layout/Navigation.css]: a real one!

Comment (jden) [inline components/Layout/Navigation.css]: missing `!important`

Comment (roseleaf) [inline components/Layout/Navigation.css]: #troll

Comment (thebritican) [inline components/Layout/Navigation.css]: ok i'm making it now.........

Comment (thebritican) [inline components/Layout/Navigation.css]: See #289

**Pull request merged**